minecraft_servers/playvenify/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+    "server_name": "playvenify",
+    "nice_name": "PlayVenify",
+    "direct_ip": "mc.playvenify.net",
+    "server_wildcards": [
+      "%.playvenify.net",
+      "%.nurserver.net"
+    ],
+    "supported_languages": [
+      "de", 
+      "en"
+    ],
+    "social": {
+      "web": "https://www.playvenify.net",

```

Comment:
We only accept pages that are online.
<img width="1196" alt="Bildschirmfoto 2023-05-10 um 21 07 25" src="https://github.com/LabyMod/server-media/assets/45363287/9dd0d5f2-baf8-484e-9b8f-3f7edc604597">